quint-assign1/quint-assign1.py

```diff
@@ -0,0 +1,218 @@
+"""
+Assignment 1: Cryptography
+Course: CS 92SI
+Name: Quint Underwood
+Date: 9 October 2015
+
+Replace this with a description of the program.
+"""
+
+def encrypt_caesar(plaintext):
+    """
+    Encrypts plaintext using a Caesar cipher.
+    Add more implementation details here.
+    """
+    shift = int(input("By how many letters do you want to shift? "))
+    encoded_phrase = ""
+    for letter in plaintext:
+        letter = chr(ord(letter) + shift)
+        if letter > 'z':
+            letter = chr(ord(letter) - 26)
+        encoded_phrase += letter
+    #print(encoded_phrase.upper())
+    return(encoded_phrase.upper())
+
+
+def decrypt_caesar(ciphertext):
+    """
+    Decrypts a ciphertext using a Caesar cipher.
+    Add more implementation details here.
+    """
+    shift = int(input("How many letters was this phrase shifted when it was encoded? "))
+    decoded_phrase = ""
+    for letter in ciphertext:
+        letter = chr(ord(letter) - shift)
+        if letter < 'a':
+            letter = chr(ord(letter) + 26)
+        decoded_phrase += letter
+    return(decoded_phrase.upper())
+
+def encrypt_vigenere(plaintext, keyword):
+    """
+    Encrypts plaintext using a Vigenere cipher with a keyword.
+    Add more implementation details here.
+    """
+    encoded_phrase = ""
+    length = len(keyword)
+    i=0
+    for letter in plaintext:
```

Comment:
You could avoid declaring `i` above as a separate counter with python's `enumerate` utility:
`for i, letter in enumerate(plaintext)`